include/seadsa/CallSite.hh

```diff
@@ -7,8 +7,8 @@
 #include "llvm/IR/Module.h"
 #include "llvm/Pass.h"
 
-// XXX: included for Cell. 
-#include "seadsa/Graph.hh" 
+// XXX: included for Cell.
```

Comment:
there are a lot of formatting changes that are intermixed with code changes. 
They should be separated into separate commits. Maybe they are -- I am looking at this
through github interface that combines all the changes. Ideally formatting would be a separate PR to make reviewing easier.

Reply:
I have my ide set to run clang-format whenever I save, I'll make a separate commit from now on

Reply:
you can fix this now by:
* checkout current `dev10`
* clang-format files you plan to change
* submit PR
* rebase your changes once the PR is accepted
* use meld as a mergetool to resolve any conflicts

Reply:
This is what I would do, but we can also leave this as is since both @caballa and myself reviewed the code as it is already